.gitignore

```diff
@@ -329,6 +329,7 @@ pip-selfcheck.json
 config/*
 !config/PUT_YOUR_WORKFLOW_RUN_CONFIGS_HERE
 !config/README.md
+!.snakemake-workflow-catalog.yml
```

Comment:
I'm wondering: why do we need a pattern to explicitly state that we want this file under version control? Do we have a pattern that excludes YAML files on a general basis? If so, why?

Reply:
nice catch! force of habit. i thought we were excluding all secret files by default. reverted back to default.